appyters/Researcher_Summary_Report_Appyter/README.md

```diff
@@ -0,0 +1,5 @@
+# Researcher Summary Report
+
+by Nasheath Ahmed
```

Comment:
This will be redundant with the authorship injected into the page through the `appyter.json`